sdk/appcenter-analytics/src/test/java/com/microsoft/appcenter/analytics/AnalyticsTransmissionTargetTest.java

```diff
@@ -436,4 +442,87 @@ public void eventPropertiesCascading() {
         expectedProperties.put("g", "7777");
         assertEquals(expectedProperties, log.getProperties());
     }
+
+    /* Property Configurator tests */
+    @Test
+    public void setCommonSchemaProperties() {
```

Comment:
You need a test where child has no parent and also no overrides to fix code coverage on the while loops (in that the common schema log is not altered and thus checking for null values)

Reply:
Ok you did something equivalent in the meantime while I was typing this.